src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -128,7 +128,7 @@ public function __sleep(): array
      *
      * @internal
      */
-    public static function throwOnRequiredClass($class, \Exception $previous = null)
+    public static function throwOnRequiredClass(string $class, \Exception $previous = null)
```

Comment:
a

Reply:
I'm preparing a PR for `@internal` methods and will pick this one up there as well.